src/sql/src/statements.rs

```diff
@@ -94,7 +94,7 @@ fn parse_string_to_value(
             }
         }
         ConcreteDataType::Timestamp(t) => {
-            if let Ok(ts) = Timestamp::from_str(&s) {
+            if let Ok(ts) = Timestamp::from_str_utc(&s) {
```

Comment:
Also respect per-session time zone settings here?

Reply:
Leave it to the other PR.